bip-0373.mediawiki

```diff
@@ -77,8 +77,8 @@ required for aggregation. If sorting was done, then the keys must be in the sort
 key the participant is providing the nonce for, followed by the BIP 341 tapleaf hash of
 the Taproot leaf script that will be signed. If the aggregate key is the taproot internal key or the
 taproot output key, then the tapleaf hash must be omitted. The plain public key must be
-the key found in the script and not the aggregate public key that it was derived from, if it was
-derived from an aggregate key.
+the output key or found in a script. It is not the internal key nor the aggregate public key that
+it was derived from, if it was derived from an aggregate key.
```

Comment:
Nit:
```The plain public key must be the taproot output key``` instead of ```The plain public key must be the output key```, matching the language of BIP-341.

Reply:
Done